app/build.gradle

```diff
@@ -45,14 +45,18 @@ android {
         debug {
             applicationIdSuffix ".debug"
         }
+        nightly {
+            initWith debug
+            applicationIdSuffix ".nightly"
+        }
```

Comment:
Let's remove the suffix. On Google Play the builds across all release tracks need to have the same package name.

Reply:
Will do.